libcaja-private/caja-icon-container.c

```diff
@@ -8740,6 +8746,16 @@ caja_icon_container_set_keep_aligned (CajaIconContainer *container,
     }
 }
 
+void
+caja_icon_container_set_lock_icons_position (CajaIconContainer *container,
+                                             gboolean lock_icons_position)
+{
+    if (container->details->lock_icons_position != lock_icons_position)
+    {
+        container->details->lock_icons_position = lock_icons_position;
+    }
+}
+
```

Comment:
Build warnings. https://travis-ci.org/github/mate-desktop/caja/jobs/764131695#L801
```
libcaja-private/caja-icon-container.c:8753:49: style: The statement 'if (container->details->lock_icons_position!=lock_icons_position) container->details->lock_icons_position=lock_icons_position' is logically equivalent to 'container->details->lock_icons_position=lock_icons_position'. [duplicateConditionalAssign]

    if (container->details->lock_icons_position != lock_icons_position)

                                                ^

libcaja-private/caja-icon-container.c:8755:49: note: Assignment 'container->details->lock_icons_position=lock_icons_position'

        container->details->lock_icons_position = lock_icons_position;

                                                ^

libcaja-private/caja-icon-container.c:8753:49: note: Condition 'container->details->lock_icons_position!=lock_icons_position' is redundant

    if (container->details->lock_icons_position != lock_icons_position)
```

Reply:
Indeed, I could ditch the test. Will do.